test/elastic-adapter.spec.js

```diff
@@ -138,7 +160,7 @@ describe('elastic source', function() {
                 });
             });
 
-            it('writes a point with a giant field', function() {
+            it('fails to write a point with a giant field', function() {
```

Comment:
Oic. Then how about a positive test that we can store max_allowed string? Maybe in the aptly named elastic-adapter-limits.spec.js